ext/standaloneusers/standaloneusers.php

```diff
@@ -77,14 +77,14 @@ function standaloneusers_civicrm_enable() {
  */
 function standaloneusers_civicrm_permission(&$permissions) {
   $permissions['access password resets'] = [
-    'label' => E::ts('Allow users to access the reset password system'),
+    'label' => E::ts('CiviCRM Standalone Users: Allow users to access the reset password system'),
   ];
-  // Concrete implementations of synthetic cms: permissions.
-  $permissions['administer users'] = [
-    'label' => E::ts('Administer user accounts'),
-    'implies' => ['view user account'],
+  // provide expected cms: permissions.
+  $permissions['cms:administer users'] = [
+    'label' => E::ts('CiviCRM Standalone Users: Administer user accounts'),
+    'implies' => ['cms: view user account'],
```

Comment:
@ufundo there is an extra whitespace here. Should be 
```suggestion
    'implies' => ['cms:view user account'],
```

Reply:
argh not again! thanks for catching